### DISCUSSION TITLE
multilingual Scripture

### DISCUSSION THREAD

**Discussion by jubinjsebastian** [ChurchApps/ChurchAppsSupport#308] [FreeShow] [answered]
It would be good to have multilingual Scripture in the app. Native language users would like to present the scripture in multilingual (English & Local language). There are thousands of Indian users who would love to have multilingual scripture in the app. Malayalam, Hindi, and Tamil is the biggest user base in India.

**Comment by vassbo** [comment, marked as answer]
Please see this 😊: https://github.com/ChurchApps/FreeShow/issues/236#issuecomment-1683423643
